script/elevate-cpanel.PL

```diff
@@ -925,24 +1030,47 @@ sub run_stage_3 ($self) {
             }
 
             # Avoids having to do preupgrade.
-            setup_answer_file();
+            setup_answer_file() if $self->should_run_leapp();
```

Comment:
why not put this in setup_answer_file?

Reply:
we can do this too, both approach work, would need to convert it to a method
and this would be more obscure while reading `run_stage_3`, this is why I decided to put the protection here as there is a single caller